NuCivic/dkan
# Make machine name not required so validation works when no title provided

Issue: https://jira.govdelivery.com/browse/CIVIC-4449

## Description

If we try to create a harvest node without title the validation fail and then show the machine name field. If you try to fill after validation the title the field machine name never is filled.


## How to reproduce

1. Go to create a Harvest Source and confirm that the 'machine-readable name" field is not visible.
2. Click 'Save' without adding any title.
3. The "machine-readable name" field will appear and will not be filled in automatically anymore

## QA Tests

- [x] Try to create a harvest node with empty title.
- [x] After show error you can't see the machine name field.
- [x] Fill title and submit node and we have the value into machine name field.
